packages/ui-components/src/components/form.tsx

```diff
@@ -792,22 +790,20 @@ function CustomSelect<
 
   const _onBlur = () => onBlur(id, selectedIndexes);
   const _onFocus = () => onFocus(id, selectedIndexes);
-  const _onChange = (_: any, data: OptionOnSelectData) => {
-    const newValue = getValue(data, multiple);
-    return onChange(enumOptionsValueForIndex<S>(newValue, enumOptions, optEmptyVal));
+  const _onChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    const newValue = multiple
+      ? Array.from(event.target.selectedOptions, option => option.value)
+      : event.target.value;
+    onChange(enumOptionsValueForIndex<S>(newValue, enumOptions, optEmptyVal));
   };
   const showPlaceholderOption = !multiple && schema.default === undefined;
 
   return (
-    <Field
-      label={labelValue(label, hideLabel)}
-      validationState={rawErrors.length ? 'error' : undefined}
-      required={required}
-    >
+    <label>
+      {labelValue(label, hideLabel)}
       <Select
+        aria-invalid={rawErrors.length > 0}
         id={id}
-        name={id}
-        multiselect={multiple}
         className='form-control'
```

Comment:
In our case, the attribute for multiple selection is `multiple`. And let's not set a class name for now.

```suggestion
        multiple={multiple}
```